v2/provisioning/nginx/README.md

```diff
@@ -0,0 +1,37 @@
+# NGINX
+
+This directory contains configuration to run an nginx server as a proxy similar to how we do it in a remote environment.
+
+Specifically has been setup to allow for running with HTTPS locally to debug issues related to this.
```

Comment:
Very similar but I think it would be clearer to say this:
```suggestion
Specifically this has been set up to allow the debugging of issues related to running with HTTPS locally.
```